sql/core.go

```diff
@@ -637,6 +637,15 @@ type AlterableTable interface {
 	ModifyColumn(ctx *Context, columnName string, column *Column, order *ColumnOrder) error
 }
 
+// AlterableDefaultsTable should be implemented by tables that can add, change, or remove the default values on their columns.
+type AlterableDefaultsTable interface {
```

Comment:
Can't this be represented by AlterableTable.ModifyColumn? If so, would prefer that to another interface

Seems like `ALTER TABLE t1 ALTER col DROP DEFAULT` should be equivalent to `ALTER TABLE t1 MODIFY col <new definition without default>`

Ditto for setting a new default

Reply:
Yeah that’s a fair point, I’ll rework it